Examples/VisionExamples/Examples/ExternalCameraViewController.swift

```diff
@@ -54,6 +60,11 @@ class FileVideoSource: ObservableVideoSource {
                     // construct `VideoSample` specifying the format of image contained in a sample buffer
                     let videoSample = VideoSample(buffer: buffer, format: .BGRA)
                     observer.videoSource(self, didOutput: videoSample)
+
+                    // Obtain camera parameters and set them each time they change
```

Comment:
nit: 
```suggestion
                    // Uncomment code below to obtain camera parameters and set them each time they change
```
?

Reply:
Not really. Just uncommenting won't get you the camera parameters. This is only a demonstration of how they can be constructed and passed.